entrypoint.sh

```diff
@@ -11,7 +11,11 @@ if [ -n "${3}" ]; then
    EXTRA_ARGS="${EXTRA_ARGS} --add-context=${3}"
 fi
 
-if [ "${4,,}" = "true" ]; then
+if [ -n "${4}" ]; then
+   EXTRA_ARGS="${EXTRA_ARGS} --with-scm-file=${4}"
+fi
```

Comment:
```suggestion
if [ -n "${4}" ]
then
   EXTRA_ARGS="${EXTRA_ARGS} --with-scm-file=${4}"
else
   EXTRA_ARGS="${EXTRA_ARGS} --with-scm"
fi
```

then drop the `--with-scm` from below?